### ISSUE TITLE
可以让大纲里的内链鼠标移上去的时候浮窗打开这个内链文件吗?

### ISSUE THREAD

**Issue by exam-ple** [guopenghui/obsidian-quiet-outline#188]
比如大纲里有个标题是bbb[[a]]ccc, 鼠标移动到这个[[a]]时, 会浮窗打开a文件, 这个能实现吗?

非常感谢

**Comment by guopenghui** [comment]
感觉这个操作不是很通用。
而且现在按ctrl加鼠标悬浮时，会弹出这一章的浮窗，和显示内链浮窗有些冲突。所以可能短期内不会做这个功能

**Comment by exam-ple** [comment]
是的, 目前鼠标移到大纲里的某一行时, 是弹出这一行所在的文件的浮窗, 但是如果能让移到内链时弹出内链的文件就太牛了. 冲突应该还好, 一行中内链所占的地方其实很少, 将触发内链的优先级放在前面应该就可以, 而行头行尾的空间以及其它非内链的地方都可以触发这一章的浮窗, 相互间应该不太妨碍

主要应用的场景是, 滚大纲肯定比滚正文快, 有时在大纲里已经找到了要看的内链了, 但要看那个内链文件, 得在大纲里点那一行, 让正文同步到那一行, 再在正文中找到要看的那个内链, 然后鼠标再靠近内链后浮窗看, 等于步骤多了好多

当然,, 目前插件已经很完善了, 这个有点像是在好的基础上追求更好的精益求精了😀
